app/helpers/media_objects_helper.rb

```diff
@@ -98,6 +98,13 @@ def display_related_item media_object
         media_object.related_item_url.collect{ |r| link_to( r[:label], r[:url]) }
       end
 
+      def display_rights_statement media_object
+        return nil unless media_object.rights_statement.present?
+        label = ModsDocument::RIGHTS_STATEMENTS[media_object.rights_statement]
+        return nil unless label.present?
+        link_to label, media_object.rights_statement
```

Comment:
maybe open in new tab?  `link_to label, media_object.rights_statement, target: 'blank'`